app/js/beacons/templates/beacons.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <ol class="breadcrumb">
+        <li class="active">Beacons</li>
+      </ol>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12" ng-controller="beaconController">
+      <form class="form-horizontal">
+        <div class="form-group">
+          <label for="beacon-address" class="col-sm-2 control-label">Address</label>
+          <div class="col-sm-10">
+            <input type="text" ng-model="beacon.address" class="form-control" id="beacon-address" placeholder="127.0.0.1">
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="beacon-token" class="col-sm-2 control-label">Token</label>
+          <div class="col-sm-10">
+            <input type="text" ng-model="beacon.token" class="form-control" id="beacon-token" placeholder="Password">
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="beacon-users" class="col-sm-2 control-label">Users</label>
+          <div class="col-sm-10">
+            <select multiple ng-model="beacon.users" class="form-control" id="beacon-users" ng-options="user.email as user.value for user in users">
+          </div>
+        </div>
+        <div class="form-group">
+          <div class="col-sm-offset-2 col-sm-10">
+            <button type="submit" ng-click="create(beacon)" class="btn btn-default">Create</button>
+          </div>
+        </div>
```

Comment:
@ngmiller can you verify an issue I'm seeing here? When I load this template into the browser (via running the lighthouse server, logging in, and clicking "Beacons" in the nav bar), this `<div class="form-group">...</div>` doesn't appear. It's just missing. Any clue?